scripts/build-docs.sh

```diff
@@ -15,8 +17,9 @@ find tmp/*/* -type d -empty -delete
 
 # flatten json files
 # jq -s 'flatten' packages/*/docs/*.json > docs/docs.json
-rm tmp/*.json
-# add first opening bracked
+# rm tmp/*.json
```

Comment:
This line is generating an error for me locally. Can we remove it? I feel like we shouldn't need it...

CC @mtuchi